charts/stable/qbittorrent/questions.yaml

```diff
@@ -10,7 +10,32 @@ questions:
 # Include{replicas1}
 # Include{podSpec}
 # Include{containerMain}
-
+                                - variable: qbitportforward
+                                  label: Gluetun Port Forwarding
+                                  schema:
+                                    additional_attrs: true
+                                    type: dict
+                                    attrs:
+                                      - variable: enabled
+                                        label: Enable Gluetun Port Forwarding Automation
+                                        description: Enable and configure Gluetun Port Forwarding Automation.
+                                        schema:
+                                          type: boolean
+                                          default: false
+                                          show_subquestions_if: true
+                                          subquestions:
+                                            - variable: QBT_USERNAME
+                                              label: qBittorrent Username
+                                              description: qBittorrent WebUI Username
+                                              schema:
+                                                type: string
+                                                default: ""
+                                            - variable: QBT_PASSWORD
+                                              label: qBittorrent Password
+                                              description: qBittorrent WebUI Password
+                                              schema:
+                                                type: string
+                                                default: ""
```

Comment:
This looks way too indented. 
Based on what you try to access on common.yaml it should be not indented at all.

Reply:
Easy fix

Reply:
Right now looks like its under .Values.workload.main.podSpec.containers.main

Reply:
I moved it up. Seemed to work where it was, passed checks at least.

Reply:
Removing indents seems to have broke it. I put them back for now.

Reply:
@stavros-k I think this is resolved.  I plagiarized nextcloud.